cmd/quadlet/main.go

```diff
@@ -194,6 +194,8 @@ func isExtSupported(filename string) bool {
 	return ok
 }
 
+var seen = make(map[string]struct{})
```

Comment:
Nit:  I would find the code easier to read if `seen` would be an argument rather than a package-global variable.  Or a comment.  Purely looking at the change, it wasn't clear why it's fixing it.